default/scripting/techs/construction/FORCE_ENERGY_STRC.focs.py

```diff
@@ -16,68 +16,68 @@
             accountinglabel="CON_TECH_ACCOUNTING_LABEL",
             priority=AFTER_ALL_TARGET_MAX_METERS_PRIORITY,
             effects=[
-                If(
+                EffectIf(
                     condition=(Value(LocalCandidate.Industry) <= Value(LocalCandidate.TargetIndustry)),
```

Comment:
could change the `condition=` to `if=` for consistency with the `else=` below perhaps?